### ISSUE TITLE
Code Incomplete

### ISSUE THREAD

**Issue by yijie21** [oppo-us-research/ActiveGAMER#2]
Hi, thank you for this great work! However, the code cannot work since there are files lost and the code is incomplete. And can I ask why in the rendering_based_planning function , the code requires the simulator to output the ground truth depth map as a hint to assist planning? Since in real scenarios, we do not have the simulators.

`in def rendering_based_planning() of active_gs_planner.py`
`depth_gt = self.sim.simulate(self.pose_conversion_slam2sim(cand_pose).detach().cpu().numpy(), no_print=True)['depth']`

**Comment by lly00412** [comment]
> Hi, thank you for this great work! However, the code cannot work since there are files lost and the code is incomplete. And can I ask why in the rendering_based_planning function , the code requires the simulator to output the ground truth depth map as a hint to assist planning? Since in real scenarios, we do not have the simulators.
> 
> `in def rendering_based_planning() of active_gs_planner.py` `depth_gt = self.sim.simulate(self.pose_conversion_slam2sim(cand_pose).detach().cpu().numpy(), no_print=True)['depth']`

This is not related to the method, it is only because of the imperfect simulation data. Some of the regions in MP3D scenes are incomplete, so we don't want the agent stuck in the open space.  You can remove these constraints in a real-world application. Or replace it by a depth sensor.

**Comment by 6666boy** [comment]
> This is not related to the method, it is only because of the imperfect simulation data. Some of the regions in MP3D scenes are incomplete, so we don't want the agent stuck in the open space. You can remove these constraints in a real-world application. Or replace it by a depth sensor.

Great work! I want to know if in this situation the agent ​​will be trapped until the end, or trapped for a period of time in between?

**Comment by lly00412** [comment]
> > This is not related to the method, it is only because of the imperfect simulation data. Some of the regions in MP3D scenes are incomplete, so we don't want the agent stuck in the open space. You can remove these constraints in a real-world application. Or replace it by a depth sensor.
> 
> Great work! I want to know if in this situation the agent ​​will be trapped until the end, or trapped for a period of time in between?

For indoor scenes, if the agent has already explored the whole room and found an open space, it will be trapped until the end; otherwise, it will come back to the room if it has not reached the maximum steps and the exploration pool is not empty.
